model/policymodel.py

```diff
@@ -5,6 +5,7 @@
 class PolicyModel(ndb.Model):
     """Class that represents a tags and their associated schedule."""
     Name = ndb.StringProperty(indexed=True, required=True)
+    DisplayName = ndb.JsonProperty(indexed=True, required=False, repeated=False)
```

Comment:
Not sure if the display name needs to be indexed. Are we looking up entities by their display name anywhere? I think only by name, right?

Reply:
Trying out this change, I got an error. Non-indexed strings no longer seem to be supported:

```
NotImplementedError: A StringProperty must be indexed. Previously setting ``indexed=False`` was allowed, but this usage is no longer supported.
```